OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -9,33 +9,344 @@
 #endregion
 
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Text;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.FileFormats;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
 
 namespace OpenRA.Mods.Common.UtilityCommands
 {
-	class ImportLegacyMapCommand : IUtilityCommand
+	public class ImportLegacyMapCommand : IUtilityCommand
```

Comment:
and when I do https://github.com/Mailaender/OpenRA/commit/3b1290a409ea5741322e1aaaf36dd5dfa234f41d I get this:
> Cannot create an abstract class